src/sentry/workflow_engine/migration_helpers/alert_rule.py

```diff
@@ -529,3 +583,45 @@ def dual_delete_migrated_alert_rule(
         RegionScheduledDeletion.schedule(instance=data_source, days=0, actor=user)
 
     return
+
+
+def dual_delete_migrated_alert_rule_trigger(
+    alert_rule_trigger: AlertRuleTrigger,
+    user: RpcUser | None = None,
+) -> None:
+    priority = (
+        DetectorPriorityLevel.MEDIUM
+        if alert_rule_trigger.label == "warning"
+        else DetectorPriorityLevel.HIGH
+    )
+    detector_trigger = get_detector_trigger(alert_rule_trigger, priority)
+    if detector_trigger is None:
+        return None
+
+    action_filter = get_action_filter(alert_rule_trigger, priority)
+    if action_filter is None:
+        return None
+
+    detector_trigger.delete()
+    action_filter.delete()
+    logger.info("Dual deleted alert_rule_trigger", extra={"alert_rule_trigger": alert_rule_trigger})
+
+    return None
+
+
+def dual_delete_migrated_alert_rule_trigger_action(
+    trigger_action: AlertRuleTriggerAction,
+    user: RpcUser | None = None,
+) -> None:
+    aarta = ActionAlertRuleTriggerAction.objects.filter(
+        alert_rule_trigger_action=trigger_action
+    ).first()
+    if aarta is None:
+        logger.error(
+            "ActionAlertRuleTriggerAction does not exist",
+            extra={"alert_rule_trigger_action_id": trigger_action.id},
+        )
+        return None
```

Comment:
thoughts on using `.get` instead of `.filter`? .get will only get 1 instance, if it has more than 1 it will throw an error. if there isn't one, it will also throw.

if we want this to be limited to 1 and can use .get; we'd be able to eliminate the `.first` chained method and the `if aarta is None:` block.